developer-docs-site/docs/reference/move.md

```diff
@@ -0,0 +1,110 @@
+
+In Move, a transaction's sender is represented by a `signer` or a verified owner of a specific account. The *signer* has the highest level of permission in Move and is the only entity capable of adding resources into an account. In addition, a module developer can require that a signer be present to access resources or modify assets stored within an account.
+
+## Comparison to other VMs
+
+| | Aptos / Move | Solana / SeaLevel | EVM |
+|---|---|---|---|
+| Data storage | Stored within the owner's account | Stored within the owner's account associated with a program | Stored within the account associated with a smart contract |
+| Parallelization | Capable of inferring parallelization at runtime within Aptos | Requires specifying within the transaction all accounts and programs accessed | Currently serial nothing in production | 
+| Transaction safety | Sequence number | Transaction uniqueness + remembering transactions | nonces, similar to sequence numbers |
+| Type safety | Module structs and generics | Program structs | Contract types |
+| Function calling | Explicit static dispatch | Static dispatch | Dynamic dispatch |
+
+## Aptos Move Features
+
+Each deployment of the MoveVM has the ability to extend the core MoveVM with additional features via an adapter layer. Furthermore, Move functions like a computer in which one can install a framework, which is similar to an operating system of a computer.
+
+The Aptos Move adapter features include:
+* Fine grained storage that allows storing many resoures within an account without increasing the gas fees associated with reading or writing to an account
+* [Tables](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/Table.move) for storing key, value data within an account at scale
+* Parallelism via [Block-STM](https://medium.com/aptoslabs/block-stm-how-we-execute-over-160k-transactions-per-second-on-the-aptos-blockchain-3b003657e4ba) that enables concurrent execution of transactions without any input from the user
+
+The Aptos framework ships with many useful libraries:
+* A [Token standard](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/Token.move) that makes it possible to create NFTs and other rich tokens without publishing a smart contract
+* A [Coin standard](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/Coin.move) that makes it possible to create typesafe Coins by publishing a trivial module
+* An [iterable Table](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/IterableTable.move) that allows for traversing all the keys within a the Table
+* A staking and delegation framework
+* A [`type_of`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/TypeInfo.move) service to identify at run-time the address, module, and struct name of a given type
```

Comment:
I would either put `at run-time` before `identify` or after everything:

service at run-time to identify the address, module, and struct name of a given type

service to identify the address, module, and struct name of a given type at run-time